Submitted By: DJ Lucas <dj_AT_linuxfromscratch_DOT_org>
Date: 2005-11-07
Initial Package Version: 1.1.3
Origin: Gentoo
Description: Fixes compilation with nptl.
Upstream Status: Unknown (uneeded for target)

--- OOo_2.0.0-orig/sal/osl/unx/process.c	2005-09-08 09:59:02.000000000 -0500
+++ OOo_2.0.0/sal/osl/unx/process.c	2005-10-19 20:41:48.000000000 -0500
@@ -507,11 +507,6 @@
 			for (i = 0; data.m_pszEnv[i] != NULL; i++)
 	     		putenv(data.m_pszEnv[i]);
 
-#if defined(LINUX)
-            /* mfe: linux likes to have just one thread when the exec family is called */
-            /*      this np function has this purpose ...                              */
-            pthread_kill_other_threads_np();
-#endif
             OSL_TRACE("ChildStatusProc : starting '%s'",data.m_pszArgs[0]);
      
 			/* Connect std IO to pipe ends */
